CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 ## Unreleased
 
+## 0.8.7 (April 18, 2019)
```

Comment:
You can get rid of this line, just leave the notes underneath the `unreleased` section.

Reply:
done